documentation/modules/rn-2.4.adoc

```diff
@@ -0,0 +1,147 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-24_{context}"]
+= {project-full} 2.4
+
+Migrate virtual machines (VMs) from VMware vSphere or {rhv-full} or {osp} to {virt} with {the-lc} {project-first}.
+
+The release notes describe technical changes, new features and enhancements, and known issues.
+
+[id="technical-changes-24_{context}"]
+== Technical changes
+
+This release has the following technical changes:
+
+.Faster disk image migration from RHV
+
+Disk images are not converted anymore using virt-v2v when migrating from RHV. This way, the migration is faster and not limited to guest operating systems that are supported by virt-v2v. link:https://github.com/kubev2v/forklift-controller/issues/403[(forklift-controller#403)]
+
+.Faster disk transfers by ovirt-imageio client (ovirt-img)
+
+Disk transfers use ovirt-imageio client (ovirt-img) instead of Containerized Data Import (CDI) when migrating from RHV to the local OCP cluster, accelerating the migration.
```

Comment:
1. backticks
2. OCP is not used in customer-facing documentation -- use either Red Hat OpenShift or OpenShift Container Platform if you mean the that specifically